airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -89,7 +90,9 @@ public static XminStatus getXminStatus(final JdbcDatabase database) throws SQLEx
     return new XminStatus()
         .withNumWraparound(result.get(NUM_WRAPAROUND_COL).asLong())
         .withXminXidValue(result.get(XMIN_XID_VALUE_COL).asLong())
-        .withXminRawValue(result.get(XMIN_RAW_VALUE_COL).asLong());
+        .withXminRawValue(result.get(XMIN_RAW_VALUE_COL).asLong())
+        .withVersion(2L)
+        .withStateType(StateType.XMIN);
```

Comment:
Should `StateType` be a commonly defined class instead of something in the Xmin POJO? I assume that `CdcState` and cursor-based state will also use this